changelog.md

```diff
@@ -1,3 +1,10 @@
+### v4.5.5
```

Comment:
Oops! I just read the "New" prefix which means minor bump. That should be "Chore" since it's not a functional change.